README.md

```diff
@@ -4,6 +4,11 @@ A client transport that uses the [URLSession](https://developer.apple.com/docume
 
 Use the transport with client code generated by [Swift OpenAPI Generator](https://github.com/apple/swift-openapi-generator).
 
+## Supported platforms and minimum versions
+ | macOS | Linux | iOS | tvOS | watchOS |
+ | :-: | :-: | :-: | :-: | :-: |
+ | ✅ 10.15+  | ✅     | ✅ 13+ | ✅ 13+ | ✅ 6+ |
+
```

Comment:
Good catch – can you please also add this to https://github.com/apple/swift-openapi-urlsession/blob/main/Sources/OpenAPIURLSession/Documentation.docc/Documentation.md?

Reply:
Do you prefer it in the header or the body?
Since there is no linux version to specify there is a trailing + if we go with `@Available`

![image](https://github.com/apple/swift-openapi-urlsession/assets/95201442/0771f96a-d3e3-4925-b78d-8c3cdd4d0c4e)

Reply:
The body is fine, similar to the README. The README and the Documentation.md follow a very similar structure, with small differences.

Reply:
Done